main.go

```diff
@@ -61,12 +66,26 @@ func getKeployVersion() string {
 	return latestTag + "-dev"
 }
 
+
 func main() {
 	if version == "" {
 		version = getKeployVersion()
 	}
-
 	fmt.Println(logo, " ")
 	fmt.Printf("%v\n\n", version)
+	isDocker := os.Getenv("IS_DOCKER_CMD")
+	if isDocker != "" {
+		Dsn = os.Getenv("Dsn")
+	}
+	//Initialise sentry.
+	err := sentry.Init(sentry.ClientOptions{
+		Dsn:              "",
+		TracesSampleRate: 1.0,
+	})
+	if err != nil {
+		log.Fatalf("sentry.Init: %s", err)
```

Comment:
this fatal will terminate the application and we shouldn't show this log to user make it info/debug/warn

Reply:
changed

Reply:
![Uploading image.png…]()